appengine-java8/spanner/src/main/java/com/example/appengine/spanner/SpannerTasksServlet.java

```diff
@@ -33,21 +32,17 @@
  * Example code for using the Cloud Spanner API. This example demonstrates all the common operations
  * that can be done on Cloud Spanner. These are:
  *
- * <p>
- *
- * <ul>
- *   <li>Creating a Cloud Spanner database.
- *   <li>Writing, reading and executing SQL queries.
- *   <li>Writing data using a read-write transaction.
- *   <li>Using an index to read and execute SQL queries over data.
- * </ul>
+ * <p><ul><li>Creating a Cloud Spanner database. <li>Writing, reading and executing SQL queries.
```

Comment:
I think closing `</li>` is required. Maybe these got formatted weird because those were missing?